forge-gui/res/cardsfolder/upcoming/faerie_fencing.txt

```diff
@@ -0,0 +1,10 @@
+Name:Faerie Fencing
+ManaCost:X B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ -X | NumDef$ -X | IsCurse$ True | SpellDescription$ Target creature gets -X/-X until end of turn. That creature gets an additional -3/-3 until end of turn if you controlled a Faerie as you cast this spell.
```

Comment:
maybe it works if you use the right SVar

Reply:
oops 🤦